package.json

```diff
@@ -5,9 +5,9 @@
     "markdownlint-cli": "^0.31.1"
   },
   "scripts": {
-    "coding-standards-check/markdownlint": "markdownlint '*.md' 'web/modules/custom/**/*.md'",
+    "coding-standards-check/markdownlint": "markdownlint '*.md' 'web/modules/custom/**/*.md' 'web/documentation/*.md'",
```

Comment:
Update this path if/when moving the documentation files, e.g to `… 'docs/*.md'` (or maybe just `'docs/'`)

Reply:
Changed